virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -102,6 +102,18 @@ As a cluster administrator, you can prevent users from enabling VM delete protec
 
 // CNV-52871 - Doc: Multi IOthread with fast storage
 * You can now xref:../../virt/managing_vms/virt-edit-vms.adoc#virt-configure-multiple-iothreads_virt-edit-vms[configure multiple IOThreads for virtual machines that use fast storage], such as SSD (solid-state drive) or NVMe (non-volatile memory express). This improves I/O performance by enabling multiple threads for disk access.
+//CNV-55965
+* You can now  xref:../../virt/managing_vms/virt-controlling-vm-states.adoc#virt-controlling-vm-states[control the states of VMs] from the tree view by right-clicking the VM and selecting the desired action from a pop-up menu. The following actions are available:
+** Start
+** Stop
+** Restart
+** Pause
+** Unpause
+** Change the state of all VMs in the selected project
```

Comment:
Here the actions that you'll see by right clicking on the vm tree view. They are the same actions that you'll see if you click on the ellipses menu in the list row.

For the namespace, we have the mentioned actions taht will be applied to all the vms underneeth  + Create action


Same actions can be performed on a folder inside the namespace. Same thing. they will be applied to all vms in the folder .
![Screenshot 2025-06-05 at 14 37 10](https://github.com/user-attachments/assets/ba52d61f-9c80-4259-a913-5e49d2e97b01)
![Screenshot 2025-06-05 at 14 37 20](https://github.com/user-attachments/assets/af2745f1-1d2d-4d39-b32c-51a30805a16a)